integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java

```diff
@@ -0,0 +1,44 @@
+package tachyon.client;
+
+import tachyon.client.file.options.InStreamOptions;
+import tachyon.client.file.options.OutStreamOptions;
+import tachyon.conf.TachyonConf;
+import tachyon.util.network.NetworkAddressUtils;
+
+public class StreamOptionUtils {
+  private StreamOptionUtils() {
+    // not intended for instantiation
+  }
+
+  public static OutStreamOptions getOutStreamOptionsWriteBoth(TachyonConf conf) {
+    return new OutStreamOptions.Builder(conf).setTachyonStorageType(TachyonStorageType.STORE)
+               .setUnderStorageType(UnderStorageType.SYNC_PERSIST).build();
+  }
+
+  public static OutStreamOptions getOutStreamOptionsWriteTachyon(TachyonConf conf) {
```

Comment:
The options look like WriteType, so perhaps we can have a factory that takes the WriteType as argument? @apc999, what do you think?